snmp/tests/test_unit.py

```diff
@@ -11,7 +11,7 @@
 from datadog_checks.base import ConfigurationError
```

Comment:
Should we add a integration/e2e test for the newly implemented resolver? or that's already covered by an existing test?

Reply:
It will be covered by the profile translation.